README.md

```diff
@@ -1 +1,49 @@
-# Awesome-Books-ES6
\ No newline at end of file
+# Awesome-Books-ES6
+
+In this project, we were able to build an application that stores a list of books added to the browser's localstorage and display their title and authors on the page. The goal of this project was to practice using classes instead of objects and arrays.
+
+##- Major languages: 
+- HTML
+- CSS
+- JavaScript
+
+### Prerequisites
+- Create a GitHub repository
+- Have VSCode or text editor of your choice installed
+- Have Git installed
+
+### Setup
+- git clone `https://github.com/asharanjith/Awesome-Books.git`
+- cd Awesome-Books
+- code .
+
+
+👤 **Author1**
+
+- GitHub: [@Linushaddai99](https://github.com/Linushaddai99)
+- Twitter: [@ShaddaiLinus](https://twitter.com/ShaddaiLinus)
+- LinkedIn: [@ShaddaiLinus](https://www.linkedin.com/in/linusshaddai/)
```

Comment:
- [ ] because this project is solo it would be better to remove info about your colleague in order to keep the readme clean and pro

Reply:
Fixed.